### PR TITLE
Minimum viable addon

### PR DESCRIPTION
**Pull request by jimkang**
This is a functional version of the add-on. It addresses the feedback from the previous PR (#1).

### CODE REVIEW

File: .github/workflows/run-addon.yml [hunk 1]
```diff
@@ -4,4 +4,43 @@ on: repository_dispatch
 
 jobs:
   Run-Add-On:
-    uses: MuckRock/documentcloud-addon-workflows/.github/workflows/run-addon.yml@v1
+    runs-on: ubuntu-latest
```
Comment (mitchelljkotler) [inline .github/workflows/run-addon.yml:7]: Is there some way to use the referenced workflow and add on the NLTK stuff?  That way you get any updates that are made to the standard workflow

File: README.md [hunk 2]
```diff
@@ -222,3 +221,12 @@ function to use.  The following are the top level keys in the object.
 * `id` - A UUID to uniquely identify this Add-On run.  It allows DocumentCloud
   to identify the run, as well as allowing the run to send back progress,
   status message and file updates.
+
```
Comment (mitchelljkotler) [inline README.md:224]: We should centralize the documentation so it doesn't get fragmented and out of date - can you replace the example read me with more of a description of how this add-on in particular works?  I think we will move the documentation somewhere else, and have all the add-ons link to it.

File: .github/workflows/run-addon.yml (hunk 1 above)
Comment (jimkang) [reply .github/workflows/run-addon.yml:7]: The problem with the referenced workflow is that it runs `python main.py`, and the NTLK stuff must be there before `main.py` is run, so I had to fork it to get that NLTK install in there.

Would it be possible to split the `run-addon.yml` into something like `set-up-addon.yml` and `run-addon.yml`? Then, add-ons that need to install something before running could run those base add-ons.

File: README.md (hunk 2 above)
Comment (jimkang) [reply README.md:224]: Good point! Having this show up in search results for general add-on setup would confuse things.

Comment (jimkang) [reply README.md:14]: Addressed in 3efa8f8.

**Pull request merged**